java/server/src/org/openqa/selenium/server/htmlrunner/HTMLTestResults.java

```diff
@@ -46,7 +46,9 @@
   private final HTMLSuiteResult suite;
 
   private static final String HEADER = "<html>\n" +
-      "<head><style type='text/css'>\n" +
+      "<head>\n"+
+      "<meta charset=\"UTF-8\">\n"+
```

Comment:
Is there a recommended quote style for attributes?  I see single and double here, double further down.